src/components/Steplist/stepitem.module.scss

```diff
@@ -0,0 +1,25 @@
+.steplist {
```

Comment:
Styles are still not proper. The distance between the header text and the rest of the content is different by 30 px then the one suggested in Figma for example.